src/Cake.GitHub.Tests/Helpers/MoqExtensions.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Threading.Tasks;
+using Moq;
+using Moq.Language;
+using Moq.Language.Flow;
+using Octokit;
+
+namespace Cake.GitHub.Tests
+{
+    internal static class MoqExtensions
+    {
+        public static IReturnsResult<TMock> ThrowsNotFoundAsync<TMock, TResult>(this IReturns<TMock, Task<TResult>> mock) where TMock : class =>
+            mock.ThrowsAsync(new NotFoundException("", HttpStatusCode.NotFound));
```

Comment:
This is not wrong, but I prefer string.Empty

Reply:
Sure, I've updated it to use string.Empty